build-system/tasks/helpers.js

```diff
@@ -817,6 +850,42 @@ async function getDependencies(entryPoint, options) {
   return Object.keys(result.metafile?.inputs ?? {});
 }
 
+/**
+ * @param {*} sourcemapsFile
+ * @param {*} options
+ * @return {*}
+ */
+function massageSourcemaps(sourcemapsFile, options) {
+  const sourcemaps = JSON.parse(sourcemapsFile);
+  sourcemaps.sources = sourcemaps.sources.map((source) => {
+    if (!source.startsWith('../')) {
+      return source;
+    }
+    return source.slice('../'.length);
+  });
+  if (sourcemaps.file) {
+    sourcemaps.file = path.basename(sourcemaps.file);
+  }
+
+  const extra = {
+    sourceRoot: getSourceMapBase(options),
+    includeContent: !!argv.full_sourcemaps,
```

Comment:
Hmm? `includeContent` isn't a property of a sourcemap.

Reply:
I'm copying what we do for Closure compiled code. See: https://github.com/ampproject/amphtml/blob/976b827181a53052d0866acd8ad7d11452cdf00a/build-system/compile/helpers.js#L44-L58

Reply:
Oh 😳, I think I made a mistake in the PR… If the intention is to remove `sourcesContent`, that should be done.

Reply:
Done. Fixing closure's code in https://github.com/ampproject/amphtml/pull/35763

Reply:
Done in both locations.